consensus/api/api.go

```diff
@@ -63,30 +62,32 @@ func (a *rpcApi) SyncProgress() (eth.SyncProgress, error) {
 	}, nil
 }
 
-func (a *rpcApi) BroadcastTx(tx ethTypes.Transaction) error {
-	syncInfo := new(tmtTypes.SyncInfo)
+func (a *consensusApi) BroadcastTx(tx ethTypes.Transaction) error {
+	if !a.isRunning() {
+		return fmt.Errorf("Consensus node is not running")
+	}
+
 	buf := new(bytes.Buffer)
 	if err := tx.EncodeRLP(buf); err != nil {
 		return err
 	}
 
-	params := map[string]interface{} {
-		"tx": buf.Bytes(),
-	}
-
-	_, err := a.client.Call("broadcast_tx_sync", params, syncInfo)
+	_, err := a.broadcastTx(nil, buf.Bytes())
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (a *rpcApi) Status() (tmtTypes.ResultStatus, error) {
-	status := new(tmtTypes.ResultStatus)
-	_, err := a.client.Call("status", map[string]interface{}{}, &status)
+func (a *consensusApi) Status() (tmtcTypes.ResultStatus, error) {
+	if !a.isRunning() {
+		return tmtcTypes.ResultStatus{}, fmt.Errorf("Consensus node is not running")
```

Comment:
error should not start with capital letter warning :) get a white background in your IDE :D